docs/docs-beta/README.md

```diff
@@ -63,15 +63,12 @@ This command starts a local development server and opens up a browser window. Mo
 To check the documentation for different issues, use the following:
 
 ```bash
-## Lints Markdown content using prettier
+## Lints all content, applies lint autofixes and prettier changes
 yarn lint
```

Comment:
i think this breaks our CI right? since we have a GH action that ran `prettier --check and eslint without fix`